guardian/amiable
# feat: Co-locate infrastructure for CD benefits

## What does this change?


This change adds the CDK defined infrastructure into this repository to enable it to be deployed alongside the app with RiffRaff.

This initially started as a copy/paste from the deploy-tools-platform repository, however I ended up adding a few hygiene tasks too to make it easier to work with. It might be easier to review commit by commit.

## How to test


- Deploy to CODE and see it work? ([it has!](https://riffraff.gutools.co.uk/deployment/view/9085d904-05ad-4740-9140-1d01a08924d2))

## How can we measure success?


No more manual updating of a Cloudformation stack!
